### DISCUSSION TITLE
Welcome to Auto-GPT-Powershell Discussions!

### DISCUSSION THREAD

**Discussion by TheCompAce** [TheCompAce/Auto-GPT-Powershell#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kallamamran** [comment]
Hello there,
I'm an old guy which has been around since computers first started to come to the regular household back in the 70-80's. I'm also a guy who wish I knew more about, for example, programming and wish I had been learning this back then, but instead... I finally turned into a "3D-graphics interested gamer".
I started as a gamer  with tennis, hockey, squash and handball on the Sanwa 9009 Video Game in the 80's qhich my father had brought home from one of his business trips.
![image](https://user-images.githubusercontent.com/25354644/232992253-c652b0d6-9181-4556-bbaa-d1c5dda461d6.png)
After this I got a Commodore 64 and that's where a real gamer was created 😋 Ikari Warriors, Frankie Goes To Hollywood (A game I never really understood, but could not stop playing), The Last Ninja, Ghost'n'Goblins, Boulder Dash, Wizball (A favorite), Cybernoid, Traz and so on...
I never owned an Amiga or Nintendo, but I had friends who did and often enjoyed my company ;)
After that my father brought home a computer I never have found out the name on. It was a very flat and small computer with buttons that was flat, some kind of touch-buttons which had pre-programmed commandos for programming. I never really got along with this though.
After this my father brought hom a 286 and the journey to the PC-world was complete. The games on 286 was not as good as on the Commodore 64, so this is where the graphics journey started.
It started with MS paint or similar. I took a pause picture on a random cartoon on the CRT-TV, overlayed a transparent paper and tracked the lines and then overlayed this on the CRT-monitor to pixelate the shapes to make a copy on the computer and was amazed about what could be done 😊
I started my 3D graphics journey through Real3D, Vista Pro, Sculpt Animate 4D jr and Lightwave3D on my friends Amiga and then made the move to the PC world through Pov Ray on my father's 386. From there I have moved through 3D Studio (DOS) to Softimage to Houdini to 3D Studio MAX, Rhino, Blender and so on...
Working with these programs to learn about 3D-graphics and at the same time following the 3D-game development via Riva, 3DFX Voodoo and so on, owning a graphics card from every generation from a 1MB Cirrus Logic graphics card up to the 3090 I have now has ben a trip and seeing how long the graphics has come from Pong to Cyberpunk2077 with path tracing is incredible.
With the latest generations of nVidia graphics cards and the introduction of CUDA I have slowly cultivated a interest in AI. This has made me land here in the end.
I have come here from Disco Diffusion (because it was a free alternative to MJ and Dall-E)-> Stable Diffusion -> ChatGPT... It has been a wild trip following the development of graphics back then to AI now and it's not over yet 😃
I'm relly curious what the future has to bring!

**Comment by kadukraemer** [comment]
Amazing idea! I would love to give it a try, please consider a MacOS version! Best wishes.

**Comment by TheCompAce** [comment]
Nice to meet you all.  

Kallamamran: Sounds like I a little younger then you. My first PC was a Computer the size of a suitcase that had a Keyboard that would attach to the front of it (so it would be "portable") It had a 8x8 inch black and green screen. My father used it to learn programming, and I used it to play Pong, Space Invaders, etc.  Then came the 286 that could play black and green games like Prince of Persia, and others.  Then came the 386, Sound Card installing, and 256 Colors, had some fun games, but seemed like the 486 was just around the corner.  Once I got one of those, then you could get better Graphic Cards, and Sound Cards (As long as you kept a list handy of what IRQs you already had used for other stuff.)  So then it was Commander Keen, Duke Nukem (the 2D and 3D), Doom, Quake, Wolfenstein 3D, and any other games I could buy the demos for $1.25 (Back when Demos where full games, and if you gave them money then you get 2 more full games)  Well as all this was going on, I was also wanting to make my own games, I had a couple books on QBasic, so I made a couple bad maze type games.  Soon though the internet would be shipped to my house as a AOL floppy.  Loaded it up and went to a few AOL forums and realized this is the place to learn. After about a month of this, I got grounded, seems the AOL number I was using was long-distance.  So after that I went back to just playing games, and slowing learning programming. Life caught up had to be a kid/teen for awhile. Then NetZero came out, and it was back to web (I wrote a program that would crash the NetZero program without terminating the Connection, so it removed the 30 minute at a time limit.)  So then I started learning Programming at a fever pace, although threw the years, I never got into the gamming side of programming, it has been more about converting information, and some game trainers.  Then I started messing around with Unity 3D, and just published a game on Steam.  It is a clone of the old DrugWar game called "Midnight Market" (side note bring up the memories of programming hours on in, with my TI-82 calculator *reaches over and rubs it gently*)  And then this whole AI stuff came out, and now I am trying to get a computer to do my job, so I can go back to playing and making video games.

kadukraemer: Nice to meet you but this project is aimed PowerShell which is for Windows only.

**Reply by Kallamamran** [reply]
Ohhhh... The Sound Blaster 😃 That's a nostalgic trip if anything. Messing around with the setting in config.sys and autoexec.bat to get things running as efficiently as possible with the limited amount of system RAM one had back then :)

**Config.sys**
DEVICE=C:\Windows\HIMEM.SYS
DOS=HIGH,UMB
DEVICE=C:\Windows\EMM386.EXE NOEMS

**Autoexec.bat**
LH C:\Windows\COMMAND\MSCDEX.EXE /D:123`
LH C:\WINDOWS\COMMAND\SMARTDRV 2048 16 /V
LH C:\MOUSE\MOUSE
SET SOUND=C:\PROGRA~1\CREATIVE\CTSND
SET BLASTER=A220 I5 D1 H5 P330 E620 T6
SET MIDI=SYNTH:1 MAP:G
SET PATH=C:\Windows;C:\
SET TEMP=C:\WINDOWS\TEMP
SET TMP=C:\WINDOWS\TEMP

Really nice to make your aquaintance! 😀

**Comment by TheCompAce** [comment]
Ahh yes good old days, one of my cheesiest PC prank was writing a loop in the Autoexec.bat and seeing how long it took someone to figure it out.

**Comment by ghost** [comment]
Hey there!

**Reply by AlelSIO** [reply]
Yo, mate)
